services/src/main/webapp/WEB-INF/service-context.xml

```diff
@@ -235,6 +236,21 @@
 	    <property name="operationsMessageSender" ref="operationsMessageSender" />
 	</bean>
 	
+	<bean id="solrUpdateJmsTemplate" class="org.springframework.jms.core.JmsTemplate">
+        <property name="connectionFactory" ref="jmsFactory" />
+        <property name="defaultDestinationName" value="activemq:queue:repository.solrupdate?asyncConsumer=true" />
+        <property name="pubSubDomain" value="false" />
+    </bean>
+    
+    <bean id="indexingMessageSender" class="edu.unc.lib.dl.services.IndexingMessageSender" >
+        <property name="jmsTemplate" ref="solrUpdateJmsTemplate" />
+    </bean>
+	
+	<bean id="indexingService" class="edu.unc.lib.dl.cdr.services.processing.IndexingService" >
```

Comment:
I think the indent characters around here are getting sort of mixed up.  Might as well leave it i guess.  We need to do a bulk conversion of all our spring context files and pom.xml files to space indented very soon.